app/scripts/framework/views/copyable.js

```diff
@@ -19,6 +19,11 @@ const Copyable = {
         const msg = clipboardTime
             ? Locale.detFieldCopiedTime.replace('{}', clipboardTime)
             : Locale.detFieldCopied;
+        if (AppSettingsModel.minimizeOnCopy) {
```

Comment:
This utility is used for different purposes. It shouldn't minimize here.

Reply:
could you please point to a better place? seemed like a good place to have this code without duplication

Reply:
Since it's supposed to copy after minimize only on the detail view, it can be for example, a method in details-view, which will call `fieldCopied` and then minimize the app.

Reply:
sounds good, only concern I have is history will not get this functionality, if this is not needed, I will make the change